docker-compose.yml

```diff
@@ -23,6 +23,22 @@ services:
     ## Only MacOSX users need --watch-poll
     command: server --dev --watch-poll
 
+  securityc:
+    build:
```

Comment:
It makes sense to modualrize all of the securityc stuff to another repo & image, then just use the public image.

Reply:
There is a `bpanel` specific config file that needs to come out of the `securityc` directory. Maybe it just lives in the root of `bpanel` with all of the other config files